Essentials/src/com/earth2me/essentials/UserData.java

```diff
@@ -497,15 +497,17 @@ public boolean isIgnoredPlayer(final String userName) {
     }
 
     public boolean isIgnoredPlayer(IUser user) {
-        return (ignoredPlayers.contains(user.getName().toLowerCase(Locale.ENGLISH)) && !user.isIgnoreExempt());
+        return ignoredPlayers.contains(user.getBase().getUniqueId().toString()) && !user.isIgnoreExempt();
     }
 
     public void setIgnoredPlayer(IUser user, boolean set) {
-        final String entry = user.getName().toLowerCase(Locale.ENGLISH);
+        String uuid = user.getBase().getUniqueId().toString();
```

Comment:
Bit confused here: I'm pretty sure UUID::toString is already in a serializable format? You just use UUID::fromString to convert it back into a UUID object.

Reply:
I mean I'd prefer we used the actual UUID type rather than masking it as a String - it's more obvious when reading source/method signatures that we're working with UUIDs as opposed to usernames.

Reply:
Oh OK I meant we're still using strings for storing in the userdata file, right? I might have just misunderstood what you said.